neurolib/utils/loadData.py

```diff
@@ -179,7 +218,7 @@ def _loadSubjectFiles(self, dsBaseDirectory, filterSubcorticalAAL2=False):
                     continue
                 for f in fnames:
                     # dirty
-                    subject = f.split("/")[-3]
+                    subject = f.split(os.path.sep)[-3]
```

Comment:
For Windows compatibility xD (found by Lena, our only Windows user :D)

Reply:
🤯 Windows🤯